<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorPopup</class>
 <widget class="QDialog" name="ErrorPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Soundbench Error Reporter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/sbicon.png</normaloff>:/icons/sbicon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: cyan;
}
QRadioButton:disabled {
	color: gray;
}
QRadioButton::indicator:disabled {
	border-image: url(:/radio/radio_disabled.png);
}
QRadioButton::indicator {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_off.png);
}
QRadioButton::indicator:hover {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_off_hover.png);
}
QRadioButton::indicator:pressed {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_pressed.png);
}
QRadioButton::indicator:checked {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_selected.png);
}
QRadioButton::indicator:checked:hover {
	width: 17px;
	height: 17px;
	border-image: url(:/radio/radio_selected_hover.png);
}
QPushButton {
	color: cyan;
	border-image: url(:/buttons/button75_idle.png);
}
QPushButton:disabled {
	color: gray;
	border-image: url(:/buttons/button75_disabled.png);
}
QPushButton:hover {
	border-image: url(:/buttons/button75_hover.png);
}
QPushButton:pressed {
	border-image: url(:/buttons/button75_pressed.png);
}</string>
  </property>
  <widget class="QGraphicsView" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGraphicsView {
	border-image: url(:/popups/error_popup.png);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Error Text</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>341</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane {
	background-color: black;
}
QTabWidget::tab-bar {
	alignment: center;
}
QTabBar::tab {
	border-image: url(:/buttons/tab_idle.png);
	width: 70px;
	height: 25px;
	margin: 6px;
	color: cyan;
	alignment: hcenter;
}
QTabBar::tab:hover {
	border-image: url(:/buttons/tab_hover.png);
}
QTabBar::tab:pressed{
	border-image: url(:/buttons/tab_press.png);
}
QTabBar::tab:selected {
	border-image: url(:/buttons/tab_select.png);
}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="actionsTab">
    <attribute name="title">
     <string>Actions</string>
    </attribute>
    <widget class="QGraphicsView" name="actionsBackdrop">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGraphicsView {
	border-image: url(:/backdrops/mainbackdrop.png);
}</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fix1Choice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>311</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Fix 1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fix2Choice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>311</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Fix 2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fix3Choice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>311</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Fix 3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fix4Choice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>311</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Fix 4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="abortChoice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>108</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Abort</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="fix5Choice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>311</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Fix 5</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fix6Choice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>311</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Fix 6</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fix7Choice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>311</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Fix 7</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="text">
      <string>Mouse over each fix to view the comments.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="infoTab">
    <attribute name="title">
     <string>Details</string>
    </attribute>
    <widget class="QTextBrowser" name="infoText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>406</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
	border-image: url(:/backdrops/mainbackdrop.png);
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Cantarell';&quot;&gt;Lorpsum Ipsum...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
